.github/workflows/nuget.yml

```
@@ -17,7 +17,9 @@ jobs:
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v4
       with:
-        dotnet-version: 8.0.x
+          dotnet-version: |
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 20: found a tab character where an indentation space is expected

(syntax-check)